src/Entity/AddressValidation/AVAddress.php

```diff
@@ -0,0 +1,73 @@
+<?php namespace Ups\Entity\AddressValidation;
+
+class AVAddress
+{
+    public $addressClassification;
+    public $consigneeName;
+    public $buildingName;
+    public $addressLine;
+    public $region;
+    public $politicalDivision2;
```

Comment:
According to the docs the Urbanization field is the PoliticalDivision3 field.
From the docs for Urbanization:
>Puerto Rico Political Division 3. Only Valid for Puerto Rico.